### PR TITLE
3.5.9 upgrade

### PR DESCRIPTION
**Pull request by cameron-morrow-toptal**

### CODE REVIEW

File: Library/RadarIO.Xamarin.Android/RadarSDKImpl.cs [hunk 1]
```diff
@@ -169,6 +169,13 @@ public Task<(RadarStatus, RadarTrip, IEnumerable<RadarEvent>)> StartTrip(RadarTr
             return handler.Task;
         }
 
+        public Task<(RadarStatus, RadarTrip, IEnumerable<RadarEvent>)> StartTrip(RadarTripOptions options, RadarTrackingOptions trackingOptions)
+        {
+            var handler = new TripCallbackHandler();
+            AndroidBinding.Radar.StartTrip(options.ToBinding(), trackingOptions.ToBinding(), handler);
```
Comment (tjulien) [inline Library/RadarIO.Xamarin.Android/RadarSDKImpl.cs:175]: `trackingOptions` can be null

File: Library/RadarIO.Xamarin.iOS/RadarSDKImpl.cs [hunk 2]
```diff
@@ -218,6 +218,23 @@ public Task<(RadarStatus, RadarTrip, IEnumerable<RadarEvent>)> StartTrip(RadarTr
             return src.Task;
         }
 
+        public Task<(RadarStatus, RadarTrip, IEnumerable<RadarEvent>)> StartTrip(RadarTripOptions options, RadarTrackingOptions trackingOptions)
+        {
+            var src = new TaskCompletionSource<(RadarStatus, RadarTrip, IEnumerable<RadarEvent>)>();
+            iOSBinding.Radar.StartTripWithOptions(options.ToBinding(), trackingOptions.ToBinding(), (status, trip, events) =>
```
Comment (tjulien) [inline Library/RadarIO.Xamarin.iOS/RadarSDKImpl.cs:224]: `trackingOptions` can be null

File: Library/RadarIO.Xamarin.Android/RadarSDKImpl.cs (hunk 1 above)
Comment (cameron-morrow-toptal) [reply Library/RadarIO.Xamarin.Android/RadarSDKImpl.cs:175]: in c# all non-primitive objects can be null. the `ToBinding()` extension method will also handle nulls

File: Library/RadarIO.Xamarin.iOS/RadarSDKImpl.cs (hunk 2 above)
Comment (cameron-morrow-toptal) [reply Library/RadarIO.Xamarin.iOS/RadarSDKImpl.cs]: added null guard

File: Library/RadarIO.Xamarin.Android/RadarSDKImpl.cs (hunk 1 above)
Comment (cameron-morrow-toptal) [reply Library/RadarIO.Xamarin.Android/RadarSDKImpl.cs]: added null guard

**Review by tjulien — APPROVED** [review]

**Pull request merged**